File: Server/SocialBicing/SocialBicing/model/Bike.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace com.mobilenik.socialbicing.model
{
    public class Bike
    {
        public int id;
        public string code;
        public DateTime creationTime;
        public int idState;
        public DateTime stateTime;
        public float latitude;
        public float longitude;
        public string address;
        public int idUser;
    }
}

File: Movil/SocialBicingAndroid/src/com/mobilenik/socialbicing/entities/Bike.java
package com.mobilenik.socialbicing.entities;

public class Bike {

	private String code;

	private int latitude;

	private int longitude;

	public Bike(int lat, int lon) {
		this.latitude = lat;
		this.longitude = lon;
	}

	public String getCode() {
		return code;
	}

	public int getLatitude() {
		return latitude;
	}

	public int getLongitude() {
		return longitude;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public void setLatitude(int latitude) {
		this.latitude = latitude;
	}

	public void setLongitude(int longitude) {
		this.longitude = longitude;
	}

}
